R/closure.R

```diff
@@ -59,8 +63,25 @@ make_rt_end_event <- function() {
         switch = 0.0,
         closure_time_end = time
       )
+
+      # check if hospitalisations are greater than threshold
+      state <- array(
+        state,
+        c(N_AGE_GROUPS, N_EPI_COMPARTMENTS, N_ECON_STRATA)
+      )
+      total_hosp <- get_hospitalisations(state)
+
+      if (total_hosp > parameters[["hospital_capacity"]]) {
+        rlang::env_poke(
+          parameters[["mutables"]], "hosp_switch", 1.0
```

Comment:
If you take the above suggestion, this becomes

```suggestion
          parameters[["mutables"]], "hosp_switch", TRUE
```

and similarly below, plus a simple change in `prepare_mutable_parameters` (and possibly elsewhere!)